build.gradle

```diff
@@ -86,7 +86,7 @@ ext.libraries = [appCompat          : 'com.android.support:appcompat-v7:' + supp
                  okhttp             : 'com.squareup.okhttp3:okhttp:3.10.0',
                  result             : 'com.github.kittinunf.result:result:1.3.0',
                  anko_commons       : 'org.jetbrains.anko:anko-commons:0.10.4',
-                 picasso            : 'com.squareup.picasso:picasso:2.71828',
+                 picasso            : 'com.squareup.picasso:picasso:2.5.2',
```

Comment:
Why? a bug?

Reply:
There was a breaking changing, but I fixed it and kept the updated version.